include/fmt/core.h

```diff
@@ -457,6 +468,9 @@ template <typename Char> class basic_string_view {
   }
 
   // Lexicographically compare this string reference to other.
+#if FMT_HAS_CONSTEXPR_CHAR_TRAITS  // C++17's char_traits::compare() is constexpr.
+  constexpr
+#endif
```

Comment:
No need to make this function `constexpr` in this PR, I think.
This can be done as needed in the following PRs.

Reply:
This change is for consistency across all uses of ``constexpr std::char_traits``.

Reply:
All I'm just saying is that this function is non-`constexpr` currently, unlike `basic_string_view` constructor or `write(const Char* value)`, so there is no need to make it `constexpr` in this PR.